cmd/minikube/cmd/docker-env.go

```diff
@@ -199,93 +109,28 @@ var dockerEnvCmd = &cobra.Command{
 		}
 
 		if dockerPath != "" {
-			out, err := tryDockerConnectivity("docker", ec)
+			out, err := daemonenv.TryDockerConnectivity("docker", ec)
```

Comment:
this feels not right that both the name has TryDockerConnectivity and we path "docker"
either the name should be TryConnectivity("docker, ec)
or it should be TryConnectivy("docker, ec)

Reply:
fixed